src/pages/api/health.ts

```diff
@@ -0,0 +1,54 @@
+import { gql } from "urql";
+import type { NextApiRequest, NextApiResponse } from "next";
+import { createServerClient } from "@/lib/create-graphq-client";
+import { logger } from "@/lib/logger";
+import { env } from "@/lib/env.mjs";
+
+enum HealthEnum {
+  DOWN = "DOWN",
+  UP = "UP",
+}
+
+enum HealthCodeEnum {
+  UP = 200,
+  DOWN = 503,
+}
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  const query = gql`
+    query test {
```

Comment:
Would use here more meaningful name which would help recognize the source, eg 
```suggestion
    query KlarnaHealthCheck {
```

Or you can use `app` query, which will also return information if the app token is still valid.